getting_started/overview.adoc

```diff
@@ -0,0 +1,14 @@
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+The following sections show how you can get started with OpenShift:
+
+- *link:setup.html[Setup]*: Prepare your physical or virtual systems to host OpenShift components.
+- *link:installation.html[Installing OpenShift]*: Install and launch OpenShift masters and nodes.
+- *link:try_it_out.html[Try It Out]*: Walk through a sample application lifecyle and other quick examples.
```

Comment:
As I mentioned,  I don't think it's good practice to create a TOC in the overview topic.